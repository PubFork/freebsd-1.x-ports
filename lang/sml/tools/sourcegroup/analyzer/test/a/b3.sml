signature A = sig
  structure B :sig val x:int end
end

structure A:A = struct
  structure B = struct val x=4 end
end

(* source sml "b1.sml"
     import structure C, D, A
     import signature A
     export functor F
     export structure F;
   source sml "bx.sml"
     import structure F
     export structure S
*)
