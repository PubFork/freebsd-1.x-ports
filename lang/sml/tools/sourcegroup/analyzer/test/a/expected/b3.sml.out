source sml "a/b3.sml"
  export structure A 
  export signature A ;

